js/packages/components/chat/OneToOne.tsx

```diff
@@ -118,14 +119,27 @@ const AppMessage: React.FC<{ message: string }> = ({ message }) => {
 	return msg ? <TypedMessage payload={msg} /> : null
 }
 
-const MessageList: React.FC<{ id: string }> = (props) => {
+const MessageList: React.FC<{ id: string; scrollToMessage?: number }> = (props) => {
 	const [{ row, overflow, flex, margin }] = useStyles()
 	const conversation = Messenger.useGetConversation(props.id)
-	if (!conversation) {
-		return <CenteredActivityIndicator />
+	const flatListRef = useRef<FlatList<any>>(null)
```

Comment:
You can use `userRef<FlatList<messenger.conversation.Entity>>(null)`

Reply:
Since this is only a list of message IDs, it's `<FlatList<string>>` (added and it works)